test/libsolidity/smtCheckerTests/blockchain_state/library_public_1.sol

```diff
@@ -0,0 +1,23 @@
+library L {
+	function l(address payable a) public {}
```

Comment:
What happens if it returns `msg.value`? Can it correctly assert that this is same as `msg.value` for `C.f(..)`?

Reply:
Nop, because there's no support at all to `delegatecall` currently, we just erase everything and report false positives.
I'll add a test with that and msg.sender.

Reply:
Also this is the same situation as external calls to code that is available at compile time, but not compiled together so you can't know for sure that the library's code is the one you're going to execute in runtime.
In the future we could try to support that
- if the `delegatecall` is to `address(this)`, like the recent sushi bug
- in the trusted mode